src/minisat/core/SolverTypes.h

```diff
@@ -168,13 +168,13 @@ class Clause {
     void calcAbstraction() {
         assert(header.has_extra);
         uint32_t abstraction = 0;
-        for (int i = 0; i < size(); i++)
+        for (unsigned i = 0; i < size(); i++)
             abstraction |= 1 << (var(data[i].lit) & 31);
         data[header.size].abs = abstraction;  }
 
 
-    int          size        ()      const   { return header.size; }
-    void         shrink      (int i)         { assert(i <= size()); if (header.has_extra) data[header.size-i] = data[header.size]; header.size -= i; }
+    unsigned int size        ()      const   { return header.size; }
+    void         shrink      (unsigned int i)         { assert(i <= size()); if (header.has_extra) data[header.size-i] = data[header.size]; header.size -= i; }
```

Comment:
Nicer indentation?

Reply:
I tried my best, I hope you like the new indentation.